docs/examples/application/expenses-spring-boot/src/test/java/com/example/springboot/HelloControllerTest.java

```diff
@@ -0,0 +1,28 @@
+package com.example.springboot;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.jupiter.api.Test;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class HelloControllerTest {
+
+    @Autowired
+    private MockMvc mvc;
+
+    @Test
+    public void getHello() throws Exception {
+        mvc.perform(MockMvcRequestBuilders.get("/").accept(MediaType.APPLICATION_JSON)).andExpect(status().isOk())
```

Comment:
This test was failing for me. For some reason the path is set to `""` not `"/"`.

This might be the cause:

https://docs.oracle.com/javaee/7/api/javax/servlet/http/HttpServletRequestWrapper.html#getServletPath--

> This method will return an empty string ("") if the servlet used to process this request was matched using the "/*" pattern.

Which is just madness

Reply:
yeah, got rid of these tests, only kept the integration tests